File: internshipmgmt_react/internshipmgmt/src/components/Resume.js
import React, { Component } from 'react'
import './Resume.css'
import {Link} from "react-router-dom";

export const Resume=()=> {

  
  
    
    return(
      <div className='res'>
        <div className='home'>
            <h3> Apply Now </h3>
            <div className="value-container">
									<span>Name</span>
                                    <input type="text" id='Name' />
                                    <br/>
                                    <br/>
                                    <span>Age</span>
                                    <input type="text" id='Age' />
                                    <br/>
                                    <br/>
                                    <span>Course</span>
                                    <input type="text" id='Course' />
                                    <br/>
                                    <br/>
                                    <span>College</span>
                                    <input type="text" id='College' />
                                    <br/>
                                    <br/>
                                    <span>Skills</span>
                                    <input type="text" id='Skills' />
                                    <br/>
                                    <br/>
                                    <button className="reg-submit" type="submit" >
								Submit
							</button>
									
								</div>
        </div>
      </div>  
    )
}

File: Internship_mgmt/Models/Resume.cs
using Microsoft.AspNetCore.Authentication;
using System.ComponentModel.DataAnnotations;

namespace Internship_mgmt.Models
{
    public class Resume
    {
        [Key]
        public  int Sno { get; set; }
        public string Name { get; set; }
        public int Age { get; set; }
        public string Course { get; set; }
        public string College { get; set; }
        public string Skills { get; set; }
    }
}
